src/components/MovieDetails/MovieDetails.js

```diff
@@ -8,57 +8,64 @@ class MovieDetails extends Component {
     constructor(props) {
         super(props);
         this.state = {
-            movieInfo: {}
+            movieInfo: {},
+            error: ''
         }
     }
    
     componentDidMount() {
         console.log("heyyyyyyyy", this.props.id);
-        // this.props.updateClickedMovie(this.props.movieInfo.id);
         fetchMovieInfo(this.props.id)
             .then(movie => this.setState({ movieInfo: movie.movie }))
-            .catch(error => this.setState({ error: 'Having trouble finding this movie right now...please try again.'} ));
-
+            .catch(() => this.setState({ error: 'Having trouble finding this movie right now...please try again.'} ));
     }
 
-    // formatOverviewLength = () => {
```

Comment:
We need to find some way of still getting this formatting overview functionality to work if the text overflows when the screen is minimized.